### PR TITLE
parameter tck

### PR DESCRIPTION
**Pull request by caton-hpg**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

#### Description:
update tck of parameter

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [x] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

### CODE REVIEW

**Review by jievince — DISMISSED** [review]

File: tests/tck/features/yield/parameter.feature [hunk 1]
```diff
@@ -209,3 +266,11 @@ Feature: Parameter
       """
     Then a SyntaxError should be raised at runtime: Direct output of variable is prohibited near `$var'
     Then clear the used parameters
+
```
Comment (czpmango) [inline tests/tck/features/yield/parameter.feature:269]: Please add some test scenarios where parameter expressions and variable expressions are mixed.

File: tests/tck/features/yield/parameter.feature [hunk 2]
```diff
@@ -7,16 +7,48 @@ Feature: Parameter
     Given a graph with space named "nba"
     Given parameters: {"p1":1,"p2":true,"p3":"Tim Duncan","p4":3.3,"p5":[1,true,3],"p6":{"a":3,"b":false,"c":"Tim Duncan"},"p7":{"a":{"b":{"c":"Tim Duncan","d":[1,2,3,true,"Tim Duncan"]}}},"p8":"Manu Ginobili", "p9":["Tim Duncan","Tony Parker"]}
 
-  Scenario: return parameters
+  Scenario: [param-test-001] without define param
```
Comment (czpmango) [inline tests/tck/features/yield/parameter.feature:10]: Add some test scenarios where parameter expressions and aggregate expressions are mixed.

File: tests/tck/features/yield/parameter.feature [hunk 3]
```diff
@@ -7,16 +7,48 @@ Feature: Parameter
...
+    When executing query:
+      """
+      RETURN $p_not_exist AS v
+      """
+    Then a SyntaxError should be raised at runtime: Direct output of variable is prohibited near `$p_not_exist'
+
+  Scenario: [param-test-002] support null
+    When executing query:
+      """
+      RETURN $p1 is not null AS v
```
Comment (czpmango) [inline tests/tck/features/yield/parameter.feature:20]: Need to add some more complex expression scenarios, such as expression nesting, type checking, runtime error, etc.

File: tests/tck/features/yield/parameter.feature (hunk 1 above)
Comment (czpmango) [reply tests/tck/features/yield/parameter.feature:269]:
```
 $var=go from "Tim Duncan" over like yield like._dst as id
 | yield $-.id+$p1+count(distinct abs($p1)+$-.id)
```

File: tests/tck/features/yield/parameter.feature (hunk 2 above)
Comment (caton-hpg) [reply tests/tck/features/yield/parameter.feature:10]: done

File: tests/tck/features/yield/parameter.feature (hunk 3 above)
Comment (caton-hpg) [reply tests/tck/features/yield/parameter.feature:20]: done

File: tests/tck/features/yield/parameter.feature (hunk 1 above)
Comment (caton-hpg) [reply tests/tck/features/yield/parameter.feature:269]: done

File: tests/tck/features/yield/parameter.feature [hunk 4]
```diff
@@ -170,42 +240,94 @@ Feature: Parameter
...
+      yield $var.id+$p1+hash(abs($p1)+$var.id) as v
       """
-    Then a SyntaxError should be raised at runtime: Direct output of variable is prohibited near `$var'
-    Then clear the used parameters
+    Then the result should be, in any order:
+      | v                                    |
+      | "Manu Ginobili1-8422829895182987733" |
+      | "Tony Parker1803925327675532371"     |
+    # aggregate expressions
+    # crash and fixed later
```
Comment (czpmango) [inline tests/tck/features/yield/parameter.feature:298]: crash?

**Review by czpmango — APPROVED** [review]

**Review by czpmango — APPROVED** [review]

**Review by Sophie-Xie — APPROVED** [review]

**Pull request merged**